protecode/util.py

```diff
@@ -181,6 +184,7 @@ def filter_and_display_upload_results(
     upload_results: typing.Sequence[UploadResult],
     cve_threshold=7,
     ignore_if_triaged=True,
+    protecode_cvss_version=CVSSVersion.V2,
```

Comment:
again, please do not include `protecode` in variable name, and do not specify default value here